futures-util/src/future/mod.rs

```diff
@@ -46,9 +45,10 @@ pub use self::err_into::ErrInto;
 pub use self::or_else::OrElse;
 pub use self::select::Select;
 pub use self::then::Then;
-pub use self::either::Either;
 pub use self::inspect::Inspect;
 
+pub use either::Either;
```

Comment:
Can you also re-export from the `futures` facade crate? Once that's done, I think we can ship this.

Reply:
Isn't there [a pre-existing export](https://github.com/rust-lang-nursery/futures-rs/blob/master/futures/src/lib.rs#L189)?

Reply:
Oh, you're right! Sorry, I that didn't occur to me.